src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/test/messages/CourtCommonPlatformMessages.kt

```diff
@@ -50,7 +65,7 @@ private fun defendant(commonPlatformHearingSetup: CommonPlatformHearingSetup) =
   """{ 
                 "id": "${commonPlatformHearingSetup.defendantId}",
                 "masterDefendantId": "${commonPlatformHearingSetup.defendantId}",
-                "pncId": ${commonPlatformHearingSetup.pnc?.let { """ "${commonPlatformHearingSetup.pnc}" """.trimIndent() } ?: "null" },
+                ${commonPlatformHearingSetup.pnc?.let { """ "pncId": "${commonPlatformHearingSetup.pnc}", """.trimIndent() } ?: ""}
```

Comment:
this needs to be how it was before - the pnc is sometimes supplied as 
```
"pncId": null,
```

and we need to replicate this in a test.

It can also be supplied as

```
"pncId": "",
```

and of course

```
"pncId": "20003/089882A",
```

Reply:
Unless of course there is another case where the pncId is not present at all....? I haven't seen that in the test setup but I might have missed it.